content/cloud/deployment-concepts.md

```diff
@@ -30,32 +41,28 @@ The Fermyon Cloud uses [Bindle](https://github.com/deislabs/bindle) to package a
 
-In the Fermyon Cloud, we host an instance of Bindle, so when you run `spin deploy`, the command will take care of:
+Deploying to the Fermyon Cloud, involves two steps:
 1. Packaging the application and upload it to the Fermyon Cloud
-2. Creating or upgrade an application, using the bindle
-
-There is no direct interaction with Bindle when using the Fermyon Cloud.
+2. Creating or upgrade an application
 
 Let's unfold each of these steps.
 
 ### 1. Packaging and Uploading an Application
 
-The first step in deploying an application is to package all the files into parcels and generate an invoice.  All of this is handled in a staging directory, which is either a temporary directory using [this implementation](https://doc.rust-lang.org/std/env/fn.temp_dir.html#platform-specific-behavior) or the staging directory defined using the `--staging-dir` option.
+The first step in deploying an application is to package all the files using the OCI specification. The Cloud Plugin will create individual layers for the files that make up the application, to optimize changes to applications on subsequent packaging and deployment.
 
-The bindle will be named using the `name` from `spin.toml`, the `version` from `spin.toml`, and a build metadata string, which is automatically generated by Spin at deployment time.
+The OCI image will be named using the `name` from `spin.toml`, the `version` from `spin.toml`, and a build metadata string, which is automatically generated by Spin at deployment time. The result is that the version of a Spin application will be like this `1.0.0-r80e5abb`.
 
-Bindles in the Fermyon Cloud always use Semantic Versioning and require major, minor, and patch version numbers to be specified. Therefore, the version in `spin.toml` has to conform to the `MAJOR.MINOR.PATCH` format, i.e. `1.0.1` is valid, `1.0` and `1` are not valid version numbers for a Spin application. The result is that the version of a Spin application will be like this `my_app_name/1.0.0+r80e5abb`.
-
-Following packaging, the bindle will be uploaded to the Fermyon Cloud. As soon as a bindle has been uploaded, it cannot be modified or deleted. This is to preserve the integrity of the immutability of bindles.
+Following packaging, the image will be uploaded to an OCI compatible registry the Fermyon Cloud.
 
 ### 2. Create or Upgrade an Application
 
-The next step in the deployment process is to create or upgrade the application.
-
-An application in the Fermyon Cloud can have multiple revisions, which are tied to channels. These concepts are derived from [Hippo](https://docs.hippofactory.dev/) an open-source Platform as a Service (PaaS) for WebAssembly. As you deploy your application both an application, a channel and a revision will be created in the Fermyon Cloud.
+The next step in the deployment process is to create or upgrade the application. An application is upgraded, if the application name as defined in `spin.toml`, already exists in the Fermyon Cloud. 
```

Comment:
This feels like it's going to lead into something but doesn't.  The previous text kind of connected this to the health check stuff so it followed a bit more naturally, but I agree we have to take out the Hippo stuff.  Maybe that is for a later rework.

Once again the dual use of "application" is confusing, but we can tackle that in a separate PR - I realise this is existing wording.

Reply:
Yeah - also a reminiscent of Hippos concept of revision, which we never adopted. I'll cut this down to something meaningful and relevant.